Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM library/julia:1.0.2
+
+RUN apt-get update && apt-get install -yq bzip2
+
+WORKDIR /workspace
+
+COPY . .
+
+CMD ["/workspace/ci/runtests.sh"]
```

Comment:
I didn't realize that TravisCI now allowed for custom Dockerfiles.

Reply:
I think this is actually a vestige of testing with Bamboo.

Reply:
Yep. I'll :fire: